docs/tutorials/02_speed.ipynb

```diff
@@ -0,0 +1,1074 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Type of hyperparameter search (search_type)</th>\n",
+       "      <td>random</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># sampled parameter settings (param_search_iters)</th>\n",
+       "      <td>20</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scale data before fitting (scale)</th>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scaler (scaler)</th>\n",
+       "      <td>StandardScaler</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction before fitting (reduce_dim)</th>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction method (dim_reducer)</th>\n",
+       "      <td>PCA</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Cross-validation strategy (cross_validator)</th>\n",
+       "      <td>KFold</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># parallel jobs (n_jobs)</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "92956168b2034bfc925f2ecc01efce8f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Initializing:   0%|          | 0/3 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Time taken: 2.5793070793151855 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "em = AutoEmulate()\n",
+    "em.setup(X, y)\n",
+    "\n",
+    "# let's see all models\n",
+    "em.print_model_names()\n",
+    "\n",
+    "# setup with fewer models\n",
+    "start = time.time()\n",
+    "\n",
+    "em.setup(X, y, model_subset=[\"sop\", \"rbf\", \"gb\"])\n",
+    "em.compare()\n",
+    "\n",
+    "end = time.time()\n",
+    "print(f\"Time taken: {end - start} seconds\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3) run fewer cross validation folds using `cross_validator` "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With larger datasets, you might initially want to set the number of folds for the cross validation to 3 instead of 5 (the default), so that there are fewer models to fit. `AutoEmulate` takes a `cross_validator` argument, which takes an scklearn cross validator or [splitter](https://scikit-learn.org/stable/api/sklearn.model_selection.html). Let's use kfold with 3 splits, which saves 2 model fits per model."
```

Comment:
```suggestion
    "With larger datasets, you might initially want to set the number of folds (`kfold`) for the cross validation to 3 instead of 5 (the default), so that there are fewer models to fit. `AutoEmulate` takes a `cross_validator` argument, which takes an scikit-learn cross validator or [splitter](https://scikit-learn.org/stable/api/sklearn.model_selection.html). Let's use kfold with 3 splits, which saves 2 model fits per model."
```